README.md

```diff
@@ -3,5 +3,8 @@ cfractal-server
 
 Before you run the server, compile cfractal
 
+    $ git clone https://github.com/KeyboardFire/cfractal
```

Comment:
I think it would be better to use `git submodule` to make sure that the version of cfractal used works with the server.